modules/gradle-plugin/src/main/kotlin/org/jetbrains/compose/storytale/plugin/JsMultiplatformTasks.kt

```diff
@@ -106,7 +106,8 @@ fun Project.createWasmAndJsStorytaleExecTask(compilation: KotlinJsIrCompilation)
     val compilationTarget = compilation.target as KotlinJsIrTarget
 
     afterEvaluate {
-        compilationTarget.browser.configureBuild(compilation)
-        compilationTarget.browser.configureRun(compilation)
+        // TODO: fix compatibility with Kotlin 2.1.20
```

Comment:
cc: @JSMonk (I didn't find a good solution yet). The APIs were removed here https://github.com/JetBrains/kotlin/commit/08ac97bd#diff-f77064ff81d3086eaa8418647a07fddf1b0b4ade938ef48119ee635f686b37f9

Reply:
fixed after rebasing